### PR TITLE
[Bulk]MS-10-13-AspNetDocs-fixes

### PR DESCRIPTION
**Pull request by Roudabeh1348**

### CODE REVIEW

**Comment by Roudabeh1348** [comment]
@Rick-Anderson The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues.
This PR includes only link fixes and does not change other content. I’d very much appreciate your timely review and feedback on the link changes I’ve suggested.

Thank you!

File: aspnet/web-pages/overview/mobile/rendering-aspnet-web-pages-sites-for-mobile-devices.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ The following example shows how to enable mobile rendering by adding a content p
     ![mobilesites-2](rendering-aspnet-web-pages-sites-for-mobile-devices/_static/image2.png)
 
 > [!NOTE]
-> To test mobile pages, you can use a mobile device simulator that runs on a desktop computer. This tool lets you test web pages as they would look on mobile devices (that is, typically with a much smaller display area). One example of a simulator is the [User Agent Switcher add-on](http://addons.mozilla.org/firefox/addon/user-agent-switcher/) for Mozilla Firefox, which lets you emulate various mobile browsers from a desktop version of Firefox.
+> To test mobile pages, you can use a mobile device simulator that runs on a desktop computer. This tool lets you test web pages as they would look on mobile devices (that is, typically with a much smaller display area). One example of a simulator is the [User Agent Switcher add-on](https://addons.mozilla.org/en-US/firefox/addon/uaswitcher/) for Mozilla Firefox, which lets you emulate various mobile browsers from a desktop version of Firefox.
```
Comment (Rick-Anderson) [inline aspnet/web-pages/overview/mobile/rendering-aspnet-web-pages-sites-for-mobile-devices.md:49]:
```suggestion
> To test mobile pages, you can use a mobile device simulator that runs on a desktop computer. This tool lets you test web pages as they would look on mobile devices (that is, typically with a much smaller display area). One example of a simulator is the [User Agent Switcher add-on](https://addons.mozilla.org/firefox/addon/uaswitcher/) for Mozilla Firefox, which lets you emulate various mobile browsers from a desktop version of Firefox.
```

File: aspnet/web-pages/overview/releases/top-features-in-web-pages-2.md [hunk 2]
```diff
@@ -460,7 +460,7 @@ To build and run the code sample:
 4. Run a mobile browser (or a mobile device emulator) and browse to *Page1.cshtml*. Notice that this time Web Pages renders the mobile version of the page. 
 
     > [!NOTE]
-    > To test mobile pages, you can use a mobile device simulator that runs on a desktop computer. This tool lets you test web pages as they would look on mobile devices (that is, typically with a much smaller display area). One example of a simulator is the [User Agent Switcher add-on](http://addons.mozilla.org/firefox/addon/user-agent-switcher/) for Mozilla Firefox, which lets you emulate various mobile browsers from a desktop version of Firefox.
+    > To test mobile pages, you can use a mobile device simulator that runs on a desktop computer. This tool lets you test web pages as they would look on mobile devices (that is, typically with a much smaller display area). One example of a simulator is the [User Agent Switcher add-on](https://addons.mozilla.org/en-US/firefox/addon/uaswitcher/) for Mozilla Firefox, which lets you emulate various mobile browsers from a desktop version of Firefox.
```
Comment (Rick-Anderson) [inline aspnet/web-pages/overview/releases/top-features-in-web-pages-2.md:463]: same as previous, remove `/en-US`

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

File: aspnet/web-pages/overview/releases/top-features-in-web-pages-2.md (hunk 2 above)
Comment (Roudabeh1348) [reply aspnet/web-pages/overview/releases/top-features-in-web-pages-2.md:463]: Removed the locale. :)

File: aspnet/web-pages/overview/mobile/rendering-aspnet-web-pages-sites-for-mobile-devices.md (hunk 1 above)
Comment (Roudabeh1348) [reply aspnet/web-pages/overview/mobile/rendering-aspnet-web-pages-sites-for-mobile-devices.md:49]: Removed the locale.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**